src/biokbase/narrative/jobs/job.py

```diff
@@ -252,31 +295,6 @@ def in_cells(self, cell_ids: List[str]) -> bool:
         else:
             return self.cell_id in cell_ids
 
-    def batch_cell_ids(self):
-        if self.batch_job:
-            child_cell_ids = [child_job.cell_id for child_job in self.children]
-            return list(set(child_cell_ids))
-        return None
-
-    @property
-    def final_state(self):
-        if self.was_terminal() is True:
-            return self.state()
-        return None
-
-    def info(self):
```

Comment:
moved to bottom of the file with other methods concerned with outputting info in the narrative